compiler/rustc_ast_lowering/src/lib.rs

```diff
@@ -2299,7 +2299,11 @@ impl<'a, 'hir> LoweringContext<'a, 'hir> {
         span: Span,
     ) -> &'hir hir::ConstArg<'hir> {
         let ct_kind = match res {
-            Res::Def(DefKind::ConstParam, _) => {
+            // FIXME(min_generic_const_args): only allow one-segment const paths for now
+            Res::Def(
```

Comment:
We shouldn't match on the res as all single segment path resolutions should start being handled in hir ty lowering instead.